lib/beaker/platform.rb

```diff
@@ -3,7 +3,7 @@ module Beaker
   # all String methods while adding several platform-specific use cases.
   class Platform < String
     # Supported platforms
-    PLATFORMS = /^(huaweios|cisco_nexus|cisco_ios_xr|(free|open)bsd|osx|centos|fedora|debian|oracle|redhat|scientific|sles|ubuntu|windows|solaris|aix|archlinux|el|eos|cumulus|f5|netscaler)\-.+\-.+$/
+    PLATFORMS = /^(huaweios|cisco_nexus|cisco_ios_xr|(free|open)bsd|osx|centos|fedora|debian|oracle|redhat_fips|redhat|scientific|sles|ubuntu|windows|solaris|aix|archlinux|el|eos|cumulus|f5|netscaler)\-.+\-.+$/
```

Comment:
@ScottGarman is there a reason that `redhat_fips` needs to be a beaker platform?

Why can't it be another platform that beaker just understands as `redhat`?

Reply:
@ScottGarman I think this is what bhg should look like instead so that beaker just treats it as a redhat platform: https://github.com/puppetlabs/beaker-hostgenerator/pull/107.  Sorry I didn't catch this sooner before I merged your bhg changes.